README.md

```diff
@@ -1,7 +1,11 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Portfolio Project desktop version 
-> this is a simple web project to display the phrase "Hello Microverse"
+> This is the website for my Microverse Portfolio Project
+
+## the link below will take you to my wesite.
+
+https://shakespear95.github.io/portfoliocomputer-/
```

Comment:
[OPTIONAL] Consider using this format `[name of your project](url of your project)`.